logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/ConversationType.kt

```diff
@@ -6,5 +6,6 @@ package com.wire.kalium.logic.data.call
  */
 enum class ConversationType {
```

Comment:
I think it would be better if this type also had the value, just like `ConversationTypeCalling.avsValue`, especially if both values should be the same, and then the `fromIntToConversationType` could be simpler and less error-prone:
```
fun fromIntToConversationType(conversationType: Int): ConversationType =
    ConversationType.values().firstOrNull { it.value == conversationType } ?: ConversationType.Unknown
```

Reply:
That value is something that AVS use, it should be kept internal in calling module